tests/x11regressions/gnomecase/application_starts_on_login.pm

```diff
@@ -0,0 +1,117 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "x11test";
+use strict;
+use testapi;
+
+#testcase 5255-1503973: Gnome: Applications starts on login 
+
+sub tweak_startupapp_menu {
+    my ($string) = @_;
+
+    wait_screen_change {
+        send_key "super";
+    };
+    type_string "settings", 1;
+    send_key "ret";
+    assert_screen "gnome-settings";
+    type_string "tweak", 1;
+    send_key "ret";
+    assert_screen "tweak-tool";
+    send_key_until_needlematch "tweak-startapp", "down";
+}
+
+sub logout_and_login {
+    my ($string) = @_;
+
+    assert_and_click "system-indicator";
+    assert_and_click "user-logout-sector";
+    assert_and_click "logout";
+    send_key "ret";
+    assert_screen "displaymanager";
+    send_key "ret";
+    type_string "$password";
+    send_key "ret";
+}
+
+sub alter_status_auto_save_session {
+    my ($string) = @_;
+
+    send_key "super";
+    type_string "settings", 1;
+    send_key "ret";
+    assert_screen "gnome-settings";
+    type_string "dconf", 1;
+    send_key "ret";
+    send_key_until_needlematch "dconf-org", "down";
+    assert_and_click "unfold";
+    send_key_until_needlematch "dconf-org-gnome", "down";
+    assert_and_click "unfold";
+    send_key_until_needlematch "dconf-gnome-evolution", "down";
+    assert_and_click "scroll-down"; #this step aim to work around screen not scroll down automate issue
+    send_key_until_needlematch "gnome-session", "down";
+    assert_and_click "auto-save-session";
+    send_key "alt-f4";
+    send_key "alt-f4";
```

Comment:
I am not sure this will work in all cases as on a slow worker the second "alt-f4" might still be catched by the first window and the second one never closes